docs/src/tutorials/symbmc.md

```diff
@@ -0,0 +1,655 @@
+
+Since a predicate can only evaluate to one of two values, there exist only two equivalence classes for `®_P`, i.e. only two symbolic states w.r.t. `P`, one is the set of all states for which `®_P` is `TRUE`, and the other is the set of all values for which `®_P` is `FALSE`. In this sense, `S // ®_P` is isomorphic to the set of all binary strings of length 1 (i.e. the set of all Booleans). In the case of `®[p1,...,pm]`, there are `2^m` different `m-tuples` with values from `{TRUE,FALSE}`, so `S // ®[p1,...,pm]` is isomorphic to the set of all binary strings of length `m`. 
+
+What is the relation between `®[p1,...,pm]` and `®_P`, where `P(s) = p1(s) /\ ... /\ pm(s)`?
+Clearly, `P(s) = TRUE <=> <<p1(s),...,pm(s)>> = <<TRUE,...,TRUE>>`. 
+Consequently, there is one equivalence class in `S // ®_P`, that is equal to
+```
+C1 = { s \ in S: P(s) = TRUE }
+```
+and one equivalence class in `S // ®[p1,...,pm]` that is equal to
+```
+C2 = { s \in S: <<p1(s),...,pm(s)>> = <<TRUE, ..., TRUE>> }
+```
+
+They are one and the same, i.e. `C1 = C2`. The difference is, that splitting `P` into `m` components `p1,...,pm` splits the other (unique) equivalence class `C \in { cl \in S // ®_P : cl # C1 }` into `2^m - 1` parts, which are the equivalence classes in `{ cl \in S // ®[p1,...,pm] : cl # C2 }`.
+
+Consequently, we can also define symbolic states with respect to a set of predicates `{p1,...,pm}`, implicitly conjoined, as elements of `S // ®[p1,...,pm]`.
+Similarly, by the subset-predicate equivalence, a symbolic state, w.r.t. `{p1,...,pm}` can be viewed as one of
+```
+p1(s) /\ p2(s) /\ ... /\ pm(s)                  \* = P(s)
+~p1(s) /\ p2(s) /\ ... /\ pm(s)                 \* \
+p1(s) /\ ~p2(s) /\ ... /\ pm(s)                 \*  |
+...                                             \*   > (as a disjunction) = ~P(s) 
+~p1(s) /\ ~p2(s) /\ ... /\ ~p{m-1}(s) /\ pm(s)  \*  |
+~p1(s) /\ ~p2(s) /\ ... /\ ~p{m-1}(s) /\ ~pm(s) \* /
+```
+
+For example, take `p1(s) = s \ in R(k)` and `p2(s) = ~I(s)`. With respect to `p1(s) /\ p2(s)`, there are two symbolic states: one corresponds to the set of all states which are both reachable and in which the invariant is violated, while the other corresponds to the set of all states, which are either not reachable, or in which the invariant holds.
+Conversely, with respect to `{p1,p2}`, there are four symbolic states: one corresponds to states which are both reachable and violate the invariant, one corresponds to states which are reachable, but which do not violate the invariant, one corresponds to states which are not reachable, but violate the invariant and the last one corresponds to states which are neither reachable, nor violate the invariant.
+
+### Symbolic traces
+Having defined symbolic states, what is then the meaning of a symbolic trace?
+In the explicit model, a _trace_ of length `k` is simply a sequence of reachable states 
```

Comment:
Would it it not make sense to introduce this when presenting explicit state checking? Or is there a reason you think it should be held off until this point?